config/crd/bases/api-gateway.consul.hashicorp.com_gatewayclassconfigs.yaml

```diff
@@ -158,9 +157,3 @@ spec:
         type: object
     served: true
     storage: true
-status:
```

Comment:
Surprised to see these kinds of changes in a minor version consumption of gRPC. What's the link?

Reply:
This is due to removing `trivialVersions=true` in the `controller-gen` command as instructed at https://book.kubebuilder.io/migration/manually_migration_guide_v2_v3.html#update-your-makefile